qiita_db/sql_connection.py

```diff
@@ -1012,6 +1014,20 @@ def execute_fetchindex(self, idx=-1):
         """
         return self.execute()[idx]
 
+    def _cmds_executor(self, cmds, func_str):
```

Comment:
Yeah, I started calling it cmds and then I change to functions, but I didn't update everything.
Sorry about that!